doc/sphinx/practical-tools/utilities.rst

```diff
@@ -890,6 +890,18 @@ line timing data:
           Chars 69 - 162 [Definition~foo0~:=~Eval~comp~i...] 0.153 secs (0.136u,0.019s)
           Chars 163 - 208 [Definition~foo1~:=~Eval~comp~i...] 0.239 secs (0.236u,0.s)
 
++ ``coqtimelog2html``
+    ::
+
+       coqtimelog2html file.v file.v.time1 [file.v.time2] > file.v.html
+
+    this command produces a HTML file displaying the original `file.v`
+    with highlights for each command indicating how much time the
+    command used compared to the slowest command in the file according
+    to the given timing files. It supports between 1 and 3 timing files.
```

Comment:
```suggestion
    Produces an HTML file displaying `file.v` highlighted to show
    how much time each command used.  It can show 1 or 2 timing files at once.
```

I don't immediately understand the part about "compared ...".  Why do I care about that--or maybe, why is that worth mentioning?  Seems to me the tool picks a scale and then displays bars based on that.

It doesn't show timing info for tactics?

What happens if there is more than one command on a line?

> It supports between 1 and 3 timing files.

IIUC it takes one or two timing files (file.v.time1 and file.v.time2).

How about including a screen shot of the sample run?  Maybe something a trifle more realistic than your test case.

Reply:
>It doesn't show timing info for tactics?

tactics are commands

>IIUC it takes one or two timing files (file.v.time1 and file.v.time2).

It can also take 3 although I have no idea what that's good for.

Reply:
>We distinguish them in the documentation, e.g. they have separate indexes. Would be nice to be clear with "... command and tactic used"

I disagree.